CONTRIBUTING.md

```diff
@@ -78,6 +78,21 @@ the above approach you have both (1) popper installed in your machine
 and (2) an environment where you can modify popper and test the 
 results of such modifications.
 
+## Running the unittests locally
```

Comment:
can we take this opportunity to move the `Keywords in issue titles`, `Branches` and `Install from source` sections to be level `##`?

Reply:
Yeah, that looks better.